### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by fredo509**
For this milestone I : 

-Update the design

### CODE REVIEW

File: src/App.css
Comment (cynthiainga) [inline src/App.css:1]: - [ ] As mentioned the design is supposed to be exactly the same as the one provided on [Zeplin](https://zeplin.io/), take a look at your design and the given design below 👇🏾 👇🏾 

Your design

Screenshot 2023-05-12 at 23 13 11

Original design

Screenshot 2023-05-12 at 23 13 35

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @fredo509, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @fredo509,

Good job working on the changes highlighted by the previous reviewer 👍🏽

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ The app runs fine as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly align the horizontal line and the input fields and button of the form element to the right end just as it's in the original design so your design looks neat and organized 👍🏽

**Your Design**
Screenshot 2023-05-12 at 11 53 55 PM

**The Original Design**
Screenshot 2023-05-12 at 11 52 03 PM

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by sumairq — APPROVED** [review]
### Hi @fredo509 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**